test/lib/assertHelpers.js

```diff
@@ -36,13 +36,13 @@ var AssertHelpers = {
             });
 
             it('report', function() {
-                assert(checker.checkString(options.input).getErrorCount() === options.errors);
+                expect(checker.checkString(options.input)).to.have.error.count.equal(options.errors);
```

Comment:
To have error count equal? It seems chai tries to emulate real language, whereas this sentence structure looks out of place